apps/reittiopas.py

```diff
@@ -0,0 +1,67 @@
+from ti700.app import TerminalApp
+from gql import gql, Client
```

Comment:
This needs to be added to `requirements.txt`

Reply:
Added